### ISSUE TITLE
Insertion tool needs to bump revision.txt for localization packages

### ISSUE THREAD

**Issue by jasonmalinowski** [dotnet/roslyn#20400]
Every time we do an insertion of our core packages, we also need to bump the revision.txt for the localization packages. Right now we're not and if things get mismatched that can result in lots of breakage.

**Comment by nguerrera** [comment]
Can we put this on the next sprint? We just spent a lot of time debugging the issue all over again. cc @tmeschter

**Comment by nguerrera** [comment]
cc @Pilchie

**Comment by tmat** [comment]
Should this rather be - kill the Roslyn localization stuff in VS repo?

**Comment by nguerrera** [comment]
Ideally, yes. But if we don't do that ASAP, then we should do this ASAP

**Comment by jasonmalinowski** [comment]
Note that this will still affect 15.8 servicing and 15.9, so we probably need to do both.

**Comment by tmeschter** [comment]
Correct; the loc work will happen in 16.0, so we still need to do this.

**Comment by Pilchie** [comment]
So, who is doing it, and when is it happening?

**Comment by shyamnamboodiripad** [comment]
cc @Shyam-Gupta
